addons/hr_skills/static/src/fields/skills_one2many/skills_one2many.js

```diff
@@ -22,6 +23,12 @@ export class SkillsListRenderer extends CommonSkillsListRenderer {
         onWillStart(async () => {
             const res = await this.orm.searchCount('hr.skill', []);
             this.anySkills = res > 0;
+            [this.user] = await this.orm.read("res.users", [user.userId], ["employee_ids"]);
+            this.IsHrUser = await user.hasGroup("hr.group_hr_user");
+            this.userSubordinates = await this.orm.search(
+                "hr.employee",
+                [["id", "child_of", this.user.employee_ids]]
+            );
```

Comment:
we can make this more efficient like this, didn't test but should work
```suggestion
            this.is_manager = await this.orm.searchCount(
                "hr.employee",
                [["id", "child_of", this.user.employee_ids],
                 ["id", "=", this.env.model.root.data.id]]
            );
```

Reply:
not possible :/ The OnWillStart is not trigger at each page changement